test/functional/long/typing-stress-e2e-specs.js

```diff
@@ -7,11 +7,14 @@ import { initSession, deleteSession } from '../helpers/session';
 chai.should();
 chai.use(chaiAsPromised);
 
+// leave the long test to Travis
+const TYPING_TRIES = process.env.TRAVIS ? 200 : 10;
+
 describe('XCUITestDriver - long tests', function () {
   this.timeout(0);
 
   let driver;
-  before(async () => {
+  before(async function () {
```

Comment:
It is a bad practice to use the fat arrow in these functions. Mocha does a lot of context manipulation, which fails if we are using lexical scoping.